app/src/main/java/org/projectbuendia/client/sync/SyncAdapter.java

```diff
@@ -511,13 +484,23 @@ private void reportProgress(int progressIncrement, @Nullable String label) {
         getContext().sendBroadcast(syncProgressIntent);
     }
 
-    private Instant getLastSyncTime(ContentProviderClient provider) throws RemoteException {
+    @Nullable
+    private Instant getLastObservationSyncTime(ContentProviderClient provider)
```

Comment:
Comment to clarify this is a time from the server's clock, not our own clock (indeed the server could return any monotonically increasing value even unrelated to time, and it might even be worth giving it a name and a type that suggests this and prevents it from ever getting confused with client time).